emission/tests/analysisTests/TestClusterPipeline.py

```diff
@@ -1,171 +0,0 @@
-import unittest
-import emission.core.get_database as edb
-import emission.analysis.modelling.tour_model.cluster_pipeline as cp
-import emission.simulation.trip_gen as tg
-import uuid as uu
-
-class ClusterPipelineTests(unittest.TestCase):
```

Comment:
Sorry I'll do this by Sunday

On Wednesday, April 20, 2016, shankari <notifications@github.com> wrote:

> In emission/tests/analysisTests/TestClusterPipeline.py
> <https://github.com/e-mission/e-mission-server/pull/263#discussion_r60515559>
> :
>
> > @@ -1,171 +0,0 @@
> > -import unittest
> > -import emission.core.get_database as edb
> > -import emission.analysis.modelling.tour_model.cluster_pipeline as cp
> > -import emission.simulation.trip_gen as tg
> > -import uuid as uu
> > -
> > -class ClusterPipelineTests(unittest.TestCase):
>
> Josh, comments on the test removal?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/263/files/599f341f0bacc3ce6e5843b5c861a0ff7710a049#r60515559>
>